README.md

```diff
@@ -30,3 +31,43 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+1. Recurrence Relation:
+
+   $T(n) = 1, \text{ if } n \leq 1$
+   
+   $T(n) = 3T\left(\frac{n}{3}\right) + n^5, \text{ when } n > 1$
+
+2. Expanding Recurrence Relation:
+
+   $T(n) = 3T\left(\frac{n}{3}\right) + n^5$
+
+   $= 3\left(3T\left(\frac{n}{9}\right) + \frac{n^5}{3}\right) + n^5$
+
+   $= 9T\left(\frac{n}{9}\right) + 3\left(\frac{n^5}{3}\right) + n^5$
+
+   which will eventually lead us to...
+
+4. Substitution:
+
+   As j grows we can assume n^5...
```

Comment:
Why?

Reply:
Sorry for the confusion I didn't add the summation that selfishout requested me to add yet... I changed it and I think it makes sense now

Reply:
Still don't understand why you can replace the sum with $n^5$.